operator/v1/types_ingress.go

```diff
@@ -660,8 +660,26 @@ const (
 
 // IngressControllerCaptureHTTPCookie describes an HTTP cookie that should be
 // captured.
-// +union
 type IngressControllerCaptureHTTPCookie struct {
+	IngressControllerCaptureHTTPCookieUnion `json:",inline"`
```

Comment:
callers should start using `IngressControllerCaptureHTTPCookieUnion` instead

Reply:
new union types should be directly embedded to `IngressControllerCaptureHTTPCookie`